velero-backup-bucket.tf

```diff
@@ -0,0 +1,10 @@
+module "s3_bucket_velero_backup" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+
+  bucket = "aeternity-velero-backup-${local.env_human}"
+  acl    = "private"
+  
+  versioning = {
+    enabled = false
```

Comment:
Enabling backup versioning as a layer of protection sounds reasonable to me, why not?